Packages/GodPackage/Sources/ProfileShareFeature/ProfileShare.swift

```diff
@@ -45,21 +104,35 @@ public struct ProfileShareView: View {
 
   public var body: some View {
     WithViewStore(store, observe: { $0 }) { viewStore in
-      VStack(alignment: .center, spacing: 24) {
+      VStack(alignment: .center, spacing: 28) {
         Text("Share Profile")
+          .font(.title3)
           .bold()
 
-        HStack(spacing: 12) {
+        HStack {
+          Spacer()
           ForEach(viewStore.shareContents, id: \.self) { content in
-            Button(action: {}) {
+            Button(action: {
+              var transaction = Transaction()
+              transaction.disablesAnimations = true
+              // TODO: workaround
+              _ = withTransaction(transaction) {
+                viewStore.send(.shareContentButtonTapped(content))
+              }
```

Comment:
import AnimationDisableTransaction

```suggestion
              viewStore.send(.shareContentButtonTapped(content),  transaction: .animationDisable)
```